### ISSUE TITLE
translate() sniff returns false negative for $obj->translate()

### ISSUE THREAD

**Issue by roborourke** [WordPress-Coding-Standards/WordPress-Coding-Standards#1535]
## Bug Description

I'm using the Google Translate API client, you get the client object and that has a `translate()` method.

The sniff `WordPress.WP.I18n.NonSingularStringLiteralDomain` catches any usage of `translate()` stating it's reserved for low level usage.

## Minimal Code Snippet

```php
<?php
class Translator {
    function translate() {
       return 'foo';
    }
}
$translator = new Translator();
echo $translator->translate();
```

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.2
| PHP_CodeSniffer version | 2.9.0
| WPCS version            | 0.14.0
| WPCS install type       | Composer project local

## Tested Against `develop` branch?
- [ ] I have verified the issue still exists in the `develop` branch of WPCS.

**Comment by roborourke** [comment]
Apologies, not sure why we’re so far behind on versions :/

**Comment by jrfnl** [comment]
No worries. When you upgrade, please have a read through [the changelogs](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/releases). Especially WPCS 1.0.0 contains some BC-breaks which you probably should be aware of.
